my-app/src/index.js

```diff
@@ -1,17 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] The Calculator component should be imported in the App.js component as stated in the requirement `Once the Calculator is ready, import it and use it in your main component (App.js).`